.github/workflows/build-release.yaml

```diff
@@ -46,14 +44,24 @@ jobs:
           tag_name: ${{ github.ref }}
           release_name: Release ${{ github.ref }}
           draft: false
-          prerelease: false
-      - name: Upload Release Asset
-        id: upload-release-asset 
+          prerelease: true
```

Comment:
Does this affect the naming schema in any way?

Reply:
Nope, I left an example of the results above